src/components/time-entry-form/TimeEntryForm.tsx

```diff
@@ -38,12 +38,12 @@ function TimeEntryForm({
     });
   };
 
-  const handleCancel = (event): void => {
+  const handleCancel = (): void => {
     event.preventDefault();
```

Comment:
Is `event.PreventDefault()` needed here? It doesn't seem like a form is being submitted.

Reply:
correcto!